.github/workflows/build.yml

```
@@ -98,3 +98,23 @@ jobs:
       - name: Setup tmate debug session
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: ["1.18", "1.19"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Install dependencies
```

Leveraging caching is always a good idea for jobs like this.

```suggestion
      - uses: actions/cache@v3
        env:
          BASE_CACHE_KEY: "${{ github.job }}-${{ runner.os }}-\
            go${{ steps.setup-go.outputs.go-version }}-"
        with:
          path: |
            ${{ steps.go-cache.outputs.dir }}
          key: "${{ env.BASE_CACHE_KEY }}\
            ${{ hashFiles('go.mod') }}-\
            ${{ hashFiles('go.sum') }}"
          restore-keys: |
            ${{ env.BASE_CACHE_KEY }}
      - name: Install dependencies
```

---

@mcdonnnj resolved with [7751db5](https://github.com/cisagov/con-pca-tasks/pull/1/commits/7751db53ea6145e573bea708aea853b28cc8aea0)